### DISCUSSION TITLE
Where else can you use it?

### DISCUSSION THREAD

**Discussion by ilfae** [Andrewthe13th/Inventory_Kamera#452] [Q&A]
I found this tool at https://genshin.hemlo.cc/partybuilder/my-characters and only use it for that.
Are there any other sites where this could be used?

**Comment by man90es** [comment]
Here's three I know about, but there's probably many more:
- https://frzyc.github.io/genshin-optimizer — characters' gear optimiser with a huge community of theorycrafters
- https://genshin-center.com — upgrades planner with other additional features
- https://seelie.me — another upgrades planner with other additional features

**Comment by paranoid945** [comment]
https://genshin.mingyulab.com/
